Assets/Scripts/UI/TabsButton.cs

```diff
@@ -0,0 +1,15 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+
+[RequireComponent(typeof(Image))]
+public class TabsButton : MonoBehaviour
```

Comment:
TabButton without the **s**, please.

Reply:
This is a conflict issue, "TabButton" is used by Unity's ShaderGraph. Potentially renamed to "TabButtonController"?

Reply:
Ouch. Well, that's fine then.